### ISSUE TITLE
How to merge cells using xlsxml()

### ISSUE THREAD

**Issue by anusha-ey** [agershun/alasql#981]
I need to merge column headers, do we have any options using alasql.
I also need to start writing from 3rd row.

**Comment by mathiasrw** [comment]
I dont think we support merging headers. AlaSQL uses https://github.com/SheetJS/js-xlsx - have a look to see if you can find any documentation.
